src/libs/DateUtils.ts

```diff
@@ -806,6 +806,19 @@ function doesDateBelongToAPastYear(date: string): boolean {
     return transactionYear !== new Date().getFullYear();
 }
 
+/**
+ * Returns a boolean value indicating whether the card has expired.
+ * @param expiryMonth month when card expires
+ * @param expiryYear year when card expires
+ */
+
+function isCardExpired(expiryMonth: number, expiryYear: number): boolean {
```

Comment:
I would request to add a clarifying comment on the start index of the month if it’s 0 or 1.